edxval/serializers.py

```diff
@@ -122,6 +124,21 @@ class Meta:  # pylint: disable=C1001, C0111
         lookup_field = "edx_video_id"
         exclude = ('id',)
 
+    def get_course_video_image_url(self, video):
```

Comment:
As per discussion with @muzaffaryousaf we will keep the implementation as it is. We will later update it once the design document is completed.